sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/ComponentDescriptor.scala

```diff
@@ -211,7 +211,7 @@ private[impl] object ComponentDescriptor {
 
           CommandHandler(grpcMethodName, messageCodec, JavaPbAny.getDescriptor, methodInvokers)
 
-        case method: SubscriptionServiceMethod =>
+        case method: PubSubServiceMethod =>
```

Comment:
better name as this ServiceMethod can be used for both subscription and publishing